src/views/product-integrations-landing/components/integrations-list/index.tsx

```diff
@@ -72,10 +73,16 @@ function IntegrationCard({ integration }: IntegrationCardProps) {
 						tags={GetIntegrationTags(integration, false)}
 						size="medium"
 					/>
-					<span className={s.viewDetails}>
-						View Details
-						{isExternalLink ? <IconExternalLink16 /> : <IconArrowRight16 />}
-					</span>
+					<StandaloneLink
```

Comment:
Since this'll introduce an extra `TAB` stop and clickable area inside of an element that's also clickable, we should just render plain text here instead of `StandaloneLink`. I believe we've needed the "contents" of a `StandaloneLink` elsewhere before? I vaguely remember @zchsh creating a one-off component for it. Maybe it was `ButtonLink`. Anyways! Maybe we can export a `StandaloneLinkContents` from `StandaloneLink`, which would handle color, size, spacing, etc.?

Reply:
Ah thanks for the tag here! We have [StandaloneLinkContents](https://github.com/hashicorp/dev-portal/blob/e25b34a5457fd9fe3c0ef0185bf970e00f0bc517/src/views/certifications/components/standalone-link-contents/index.tsx) from Certifications work. If we're seeing another use for it, +1 to maybe taking this opportunity to export `StandaloneLinkContents` from `StandaloneLink` 👍 Hover effects could get tricky though (Robust but maybe-too-much-JS `useHover`? Simpler but maybe-more-brittle `@nest` in CSS? etc etc)... but if we don't need hover effects, then maybe a non-issue!

Reply:
Which hover effects? I'd think we could accomplish them by passing a `className` to `StandaloneLinkContents`.

Reply:
Yeah that's true, exposing a `className` prop could definitely work! I think we'll still need to use `@nest` in some way, or have a nested CSS structure, since we'd need `StandaloneLinkContents` to respond to a `:hover` state outside the element itself (specifically we'll need it to respond to when the `<a />` element rendered in `StandaloneLink` is hovered, I think we need an underline on hover right?).

One potential downside is that then any consumer of `StandaloneLinkContents` would maybe have to re-implement the hover styles.... where say an `isHovered` prop on `StandaloneLinkContents`, and the consumer using a `useHover` to drive that would mean we could build in the "hovered" styles to `StandaloneLinkContents` directly... at a cost of course! I'd think that cost isn't worth it, especially for now, since the hover effect seems to vary based on context anyways 👍

Reply:
Thank you! `StandaloneLinkContents` looks perfect! I'll look into these notes re: hover state complexities and come back with questions or a commit to talk through!

Reply:
I see we've also implemented this in the try-hcp-callout. I'm contemplating implementing a unique case here and then writing a ticket to uplift and unify these three instances! Will report back after digging in a bit.

Reply:
+1 on not having to do the uplift & unify here; i for one would be fine with another unique case in the context of this PR. i'd be happy to pick up that ticket as a follow-up since i imagine it'd have a pretty clear scope 💯 🙇

Reply:
Made a [follow up ticket here ](https://app.asana.com/0/1202097197789424/1204167616054151)and linked to it in the comment!